Makefile

```diff
@@ -19,8 +19,8 @@ SHELL = /bin/bash
 .PHONY: all version program path deps run race generate build clean test gofmt yamlfmt format docs rpmbuild mkdirs rpm srpm spec tar upload upload-sources upload-srpms upload-rpms copr
 .SILENT: clean
 
-SVERSION := $(shell git describe --match '[0-9]*\.[0-9]*\.[0-9]*' --tags --dirty --always)
-VERSION := $(shell git describe --match '[0-9]*\.[0-9]*\.[0-9]*' --tags --abbrev=0)
+SVERSION := $(or $(SVERSION),$(SVERSION),$(shell git describe --match '[0-9]*\.[0-9]*\.[0-9]*' --tags --dirty --always))
```

Comment:
Because I'm stupid, and I can't find the Makefile docs, why does $(SVERSION) appear twice?

` or, $(SVERSION,$(SVERSION) `
?